### ISSUE TITLE
Support K-9 Mail for icon requests.

### ISSUE THREAD

**Issue by PermaFreez** [Delta-Icons/android#641]
I just started using your iconpack and I like it very much, however some apps I use are still not supported. This is fine as you have a very good request system, however this doesn't support my mail client of choice, K-9 mail. I think this is an issue on your side, rather than theirs.

**Comment by flameshikari** [comment]
@PermaFreez hello! I've installed K-9 for testing purposes. The app shows in the list as you see in the picture below. I've sent an icon request via the app and it showed up in Delta inbox. Everything seems to work fine. It must be something on your side. 🤔

![image](https://github.com/Delta-Icons/android/assets/17804422/eed0cc15-0cab-4109-8917-822a800a9137)

**Comment by PermaFreez** [comment]
This is indeed very strange, because when I installed other email apps (Gmail, Outlook) they did show up. Even after reinstalling K-9 from both F-Droid and Aurora, from where I installed both Gmail and Outlook it didn't show up. I still have no idea of what's going on here, but since this problem only exists on my side (probably has to do with the degoogled custom ROM I'm using), I think I will close this thread.
